### PR TITLE
Rename Hidden tab in inventory to Staked

### LINKED ISSUE #90: Rename `Hidden` tab in inventory to `Staked`
### Discussed in https://github.com/TreasureProject/treasure-marketplace/discussions/89

<div type='discussions-op-text'>

<sup>Originally posted by **wyze** January  5, 2022</sup>
The only way an NFT goes to Hidden is when it is Staked. We don't have a Hide feature, so it is a little misleading.</div>

### PR DESCRIPTION
**Pull request by 0xkarel**
Quick fix to resolve #90.

### CODE REVIEW

**Comment by wyze** [comment]
I think you need to check usage of `section` variable on that page. Should be updated here at least: https://github.com/TreasureProject/treasure-marketplace/blob/9e4b29642551e33b48998bec37b6bbcc71998040/src/pages/inventory/%5B%5B...section%5D%5D.tsx#L589

**Comment by 0xkarel** [comment]
Oops! Resolved.
> I think you need to check usage of `section` variable on that page. Should be updated here at least:
> 
> https://github.com/TreasureProject/treasure-marketplace/blob/9e4b29642551e33b48998bec37b6bbcc71998040/src/pages/inventory/%5B%5B...section%5D%5D.tsx#L589

File: src/pages/inventory/[[...section]].tsx [hunk 1]
```diff
@@ -547,7 +547,7 @@ const Inventory = () => {
   const [data, totals, updates, emptyMessage] = useMemo(() => {
     const empty: Record<string, NonNullable<Nft["listing"]>> = {};
     const {
-      hidden = [],
+      staked = [],
```
Comment (wyze) [inline src/pages/inventory/[[...section]].tsx:550]: In order for this change to work, you'd have to change the GraphQL query. I think for now we can keep this as hidden.

```suggestion
      hidden = [],
```

File: src/pages/inventory/[[...section]].tsx [hunk 2]
```diff
@@ -586,8 +586,8 @@ const Inventory = () => {
     );
 
     switch (section) {
-      case "hidden":
-        return [hidden, totals, updates, "No hidden listings 🙂"] as const;
+      case "staked":
+        return [staked, totals, updates, "No staked listings 🙂"] as const;
```
Comment (wyze) [inline src/pages/inventory/[[...section]].tsx:590]: Let's change this back so we don't have to change the GraphQL query.

```suggestion
        return [hidden, totals, updates, "No staked listings 🙂"] as const;
```

**Review by wyze — CHANGES_REQUESTED** [review]
Almost there, but not quite!

**Review by wyze — APPROVED** [review]
Awesome, thanks @0xkarel!

**Comment by wyze** [comment]
@all-contributors please add @0xkarel for code

**Pull request merged**